app_dart/lib/src/request_handlers/push_build_status_to_github.dart

```diff
@@ -89,12 +89,12 @@ class PushBuildStatusToGithub extends ApiRequestHandler<Body> {
   Future<void> _updatePRs(RepositorySlug slug, String status, DatastoreService datastore) async {
     final GitHub github = await config.createGitHubClient(slug);
     final List<GithubBuildStatusUpdate> updates = <GithubBuildStatusUpdate>[];
-    await for (PullRequest pr in github.pullRequests.list(slug)) {
+    await for (PullRequest pr in github.pullRequests.list(slug, base: config.defaultBranch)) {
       final GithubBuildStatusUpdate update = await datastore.queryLastStatusUpdate(slug, pr);
       if (update.status != status) {
         log.debug('Updating status of ${slug.fullName}#${pr.number} from ${update.status} to $status');
         final CreateStatus request = CreateStatus(status);
-        request.targetUrl = 'https://ci.chromium.org/p/flutter/g/engine/console';
+        request.targetUrl = 'https://ci.chromium.org/p/flutter/g/${slug.name}/console';
```

Comment:
This won't work for framework, but after the migration to autogenerated builders in that repo it will (ETA 1 week)